libs/server/ACL/ACLParser.cs

```diff
@@ -36,10 +38,8 @@ class ACLParser
         /// <exception cref="ACLUnknownOperationException">Thrown if the given operation does not exist.</exception>
         public static User ParseACLRule(string input, AccessControlList acl = null)
         {
-
             // Tokenize input string 
-            Regex regex = new Regex("\\s+");
-            string[] tokens = regex.Split(input.Trim());
+            string[] tokens = input.Trim().Split(WhitespaceChars);
```

Comment:
I don't think this is a valid substitution for that regex. `Split()` will add empty entries to token array, if it finds multiple whitespace characters in a row (e.g., `"   "`).

Reply:
Let's trim and drop them then :+1: Good catch, thanks!
```suggestion
            string[] tokens = input.Trim().Split(WhitespaceChars, StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);
```